### DISCUSSION TITLE
Changing Dock Position

### DISCUSSION THREAD

**Discussion by amgschnappi** [elementary/triage#222] [Polls]
On some monitors, the vertical estate is at premium. While there is abandent horizontal space.
Is it possible to move the Dock to Left vertically?

**Comment by jeremypw** [comment]
There is a very small target on the current dock which when secondary-clicked gives a "Preferences" option leading a to a dialog that includes changing the orientation.  You can also change the settings manually in "net.launchpad.plank.docks.dock1".  It would be good if a vertical orientation option were continued in the new dock although I guess it could add some complications to the window manager and wingpanel.   I always have a vertical dock at the moment.
